src/contexts/collectionEditor/CollectionEditorContext.tsx

```diff
@@ -1,28 +1,28 @@
 import { createContext, memo, ReactNode, useCallback, useContext, useMemo, useState } from 'react';
 import { arrayMove } from '@dnd-kit/sortable';
 import { DragEndEvent } from '@dnd-kit/core';
-import { EditModeNft } from 'flows/shared/steps/OrganizeCollection/types';
+import { EditModeNft, StagingItem } from 'flows/shared/steps/OrganizeCollection/types';
```

Comment:
most of the changes in this file are to change the `EditModeNft` type to `StagingItem`